packages/ds-healthcare-gov/docs/src/pages/components/MultiInputDateField/_MultiInputDateField.docs.scss

```diff
@@ -1,39 +1,18 @@
-/*
-Date field
-
-@uswds https://designsystem.digital.gov/components/form-controls/#date-input
-
-Three text fields are the easiest way for users to enter most dates.
-
-Markup: datefield.example.html
-
-Style guide: components.date-field
-*/
-
-/*
-`<DateField>`
-
-@react-example DateField.example.jsx
-
-@react-props DateField.tsx
-
-Style guide: components.date-field.react
-*/
-
 /*
 ---
 
 ### When to use
 
-- Use this format for most dates, particularly memorized dates.
+- This date input option isn't recommended.
+- Use only when functionality/back-end systems can't support a single date field entry or if you must collect only month and year.
```

Comment:
This was how it was written in Confluence, however we don't currently support the collection of month and year. Should I rephrase to:
> Use only when functionality/back-end systems can't support a single date field entry.

Reply:
Yeah, that's fine. I think that sentence could use some work, but this is what was approved.